deepchem/feat/materials_featurizers.py

```diff
@@ -0,0 +1,178 @@
+"""
+Featurizers for inorganic crystals.
+"""
+
+import numpy as np
+
+from deepchem.feat import Featurizer
+from deepchem.utils import pad_array
+
+from matminer.featurizers.composition import ElementProperty
+
+from pymatgen import Composition, Structure
```

Comment:
Can you move these imports into the functions/methods where they are used? DeepChem has a system for handling optional dependencies (which we need to document...). Check out https://github.com/deepchem/deepchem/blob/master/deepchem/utils/rdkit_util.py for an example of how to do this